doc/source/commands/communication.rst

```diff
@@ -143,3 +143,68 @@ The receiving CPU will use :attr:`CORE:MESSAGES` to access its message queue::
   } ELSE {
     PRINT "Unexpected message: " + RECEIVED:CONTENT.
   }
+
+.. _connectivityManagers:
+
+Connectivity Managers
+---------------------
+
+.. note::
+
+    .. versionadded:: v1.0.2
+        The concept of selectable connectivity managers was added after KSP
+        introduced a stock communications system (CommNet). kOS was updated to
+        support both CommNet and RemoteTech. Other mods may be supported or
+        provide their own support in the future.
+
+kOS can implement communications over a variaty of connectivity configurations.
+We refer to these options as "Connectivity Managers." You can slect the active
+manager from the :ref:`kOS section of KSP's Difficulty Settings<settingsWindow>`.
+If the currently selected manager no longer is available, or if a new manager
+becomes available, you will be prompted with a dialog box to select the manager
+you want to use.
+
+.. object:: Connectivity Managers
+
+    By default kOS supports the following list of managers. If a manager is not
+    currently available (because the required mod isn't installed, or the system
+    is disabled in the settings) it will not be shown in lists of available managers.
+
+    StockConnectivityManager
+      This manager will ignore all connectivity limitations.  Connections
+      between vessels, home, and control always show as being connected. This
+      is the equivalent of setting ``CONFIG:RT`` to ``False`` under the former
+      system. It is possible kOS will be unable to use all features if this
+      manager is selected while communication limitations are enforced by
+      another mod or a setting.
+    CommNetConnectivityManager
+      This manager will use KSP's stock CommNet implementation to monitor
+      connections.  CommNet has limitations on updating connections for vessels
+      which are not the active vessel.  The best way to ensure that a connection
+      is updated is to include a relay antenna on one or both of the vessels.
+    RemoteTechConnectivityManager
+      This manager will use the RemoteTech mod to monitor connections. It will
+      only be available if RemoteTech is installed.  You can access more
+      detailed information and methods using the :ref:`RemoteTech Addon<remotetech>`
+
```

Comment:
Perhaps somewhere we need to mention the fact that the current RemoteTech implementation doesn't just ignore CommNet, but in fact *requires that you disable* CommNet because they step on each other's toes.  Just selecting RemoteTechConnectivityManager alone doesn't make things work if you happened to have picked "hard mode" when starting a career and just accepting the default "hard mode" options.  (The default turns on parts of CommNet, and thus steps on RT's toes.  To use the current RT that hasn't yet been fully integrated into CommNet, you have to go in to the options and turn all of CommNet *off*.  If they're both on, it messes up, and having CommNet turn on when you didn't realize it was on is easy for a player to do - because it comes on by default with hard mode.  Granted, this is technically an entirely RT issue, not kOS, but if we're going to say that we "work" with RT it's going to affect our users too so maybe we should mention it as well in our docs.)

I'd like to pro-actively stop the complaints that might otherwise come in about how "our" mod is broken.